src/controllers/GameController.js

```diff
@@ -406,7 +405,7 @@ router.get('/live', async (req, res) => {
         const game = await Game.findOne({ status: 'active' }).exec();
 
         if (!game) {
-            return res.render('layouts/liveGame', { game: null, noActiveGame: true });
+            return res.render('layouts/liveGame', { socketConfig: socketConfig, game: null, noActiveGame: true, infoBannerMessage });
```

Comment:
This was the error why when no game was started and you load the live game and then sztart the game the live game was not updating

Reply:
good catch